daemons/fenced/fenced_commands.c

```diff
@@ -144,7 +144,8 @@ get_action_delay_max(stonith_device_t * device, const char * action)
 }
 
```

Comment:
main occurrence of victim is in this file but do we intend to address that in general?
did a quick grep and it doesn't look impossible.